src/components/ReportViewer/report-item.tsx

```diff
@@ -123,6 +124,10 @@ const ReportItem = (props: PropsWithChildren<ReportItemProps>) => {
             {...other}
         />
     );
+}
+
+ReportItem.defaultProps = {
+    className: undefined,
 };
```

Comment:
Should be kept ?

Reply:
To delete, it's deprecated now  
https://github.com/facebook/react/pull/16210